src/functions.js

```diff
@@ -126,16 +126,27 @@ defineFunction("\\sqrt", {
     };
 });
 
-// Some non-mathy text
-defineFunction("\\text", {
+// Non-mathy text, possibly in a font
+var textFunctionStyles = {
+    "\\text": undefined, "\\textrm": "mathrm", "\\textsf": "mathsf",
+    "\\texttt": "mathtt", "\\textnormal": "mathrm", "\\textbf": "mathbf",
+    "\\textit": "textit", "\\textsc": "textsc",
```

Comment:
Nevermind... I see why.  `.mathit` sets `font-family: KaTeX_Math;`.  Sorry for all of the silly questions about my own code.  :(